api/spec/roles-list.spec.js

```diff
@@ -0,0 +1,144 @@
+const request = require("request");
+
+require("dotenv").config();
+const API_SERVER = process.env.API_SERVER;
+
+describe("Server", () => {
+    var server;
+
+    beforeAll(() => {
+        server = require("../");
+    });
+
+    describe("GET /roles", () => {
+        var data = {};
+        beforeAll((done) => {
+            request.get(API_SERVER + "/roles", (error, response, body) => {
+                data.status = response.statusCode;
+                data.body = body;
+                done();
+            });
+        });
+
+        it("Status 200", () => {
+            expect(data.status).toBe(200);
+        });
+
+        it("matches objects with the expect key/value pairs", function () {
```

Comment:
this tests states it should be testing key/value pairs but the tests actually only tests for the values?